### PR TITLE
[Dragboard] Performance and stability

### PR DESCRIPTION
**Pull request by thyttan**

### CODE REVIEW

File: apps/dragboard/lib.js [hunk 1]
```diff
@@ -1,12 +1,10 @@
-//Keep banglejs screen on for 100 sec at 0.55 power level for development purposes
-//Bangle.setLCDTimeout(30);
-//Bangle.setLCDPower(1);
-
 exports.input = function(options) {
+  "ram";
```
Comment (gfwilliams) [inline apps/dragboard/lib.js:2]: Please can you take this one out? Adding `"ram"` to everything doesn't always help - this function is only called *once* when the keyboard is first to be displayed, so really it's just using a whole bunch of RAM (while keyboard is displayed *and* afterwards until a new app is loaded) for no reason

The best thing is to do is figure out which functions get called a lot or have to be faster and tag those with `"ram"` - I can see that had been done a bit already but those mentions have been removed now?

I can imagine that `drag: function(event) {` could really benefit with being tagged with `"ram"` - that might provide you with the performance you're after?

Comment (thyttan) [reply apps/dragboard/lib.js:2]: Thank you, will look into that.

Comment (thyttan) [reply apps/dragboard/lib.js]: I removed that one and added "ram" to three sub-functions instead.

**Comment by gfwilliams** [comment]
Great, thanks!

**Pull request merged**